friends/.gitignore

```diff
@@ -0,0 +1,23 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
+
+# dependencies
+/node_modules
```

Comment:
Having to manually enter all the information into the input fields in order to delete the entry isn't very user friendly. You can either add a button for each entry that would delete the entry, or you can have an onClick on the div that would populate the fields for you.